proto/spike-service/spike.proto

```diff
@@ -33,4 +34,25 @@ message ValidateTokenResponse {
     bool valid = 1;
     repeated string scopes = 2;
     string message = 3;
+    string alias = 4;
+    User user = 5;
+}
+
+message ValidateAuthCodeTokenRequest {
+    string token = 1;
+    string audience = 2; //optional
+}
+
+message ValidateAuthCodeTokenResponse {
```

Comment:
I'd suggest writing here the meaning of the fields returned. 
I assume message is error message, alias is the system's alias? user is the userID signed in the token?